indico/modules/events/editing/templates/emails/comment_notification.txt

```diff
@@ -1,11 +1,19 @@
 {% extends 'events/editing/emails/base_email.txt' %}
 
-{% block subject -%}{% trans %}New comment{% endtrans %}{%- endblock %}
+{% block subject -%}
+{% if programme_code -%}
+    {% trans %} New comment on contribution {{contribution_id}} with programme code {{programme_code}}{% endtrans %}
+{%- else -%}
+    {% trans %} New comment on contribution {{contribution}}{% endtrans %}
```

Comment:
```suggestion
    {% trans %} New comment on "{{contribution.title}}"{% endtrans %}
```
Also, is the contribution being passed to the template? Or was this a typo of `contribution_id`?